tessellation/src/path_stroke.rs

```diff
@@ -660,6 +660,44 @@ impl<'l, Output: 'l + GeometryBuilder<Vertex>> StrokeBuilder<'l, Output> {
                 }
             }
 
+            LineJoin::Bevel => {
+            
+                // First we take our initial vertex for compute our triangle
+                let current_vertex = add_vertex!(
+                    self,
+                    Vertex {
+                        position: self.current,
+                        normal: normal,
```

Comment:
I think that the normal of this vertex might not be equal to `normal` (which is the average normal of the join), but rather a vector perpendicular to `prev_tangent`.